lib/testScenario/postmanCollectionRunnerClient.ts

```diff
@@ -371,6 +371,7 @@ export class PostmanCollectionRunnerClient implements TestScenarioRunnerClient {
       raw: JSON.stringify(body, null, 2),
     });
     this.addAuthorizationHeader(item);
+    console.log(armTemplate.outputs);
```

Comment:
Will output include sensitive data?

Reply:
Updated. It's just a debug log. I remove it.